src/common/timer_defaults.h

```diff
@@ -1,19 +1,18 @@
 //----------------------------------------------------------------------------//
 // timer_defaults.h - default values for timers
-#ifndef _TIMER_DEFAULTS_H
-#define _TIMER_DEFAULTS_H
+#pragma once
 
-#define TIM_BASE_CLK_MHZ 84
+enum {
+    TIM_BASE_CLK_MHZ = 84,
 
-//timer defaults
-//50000 us
-#define TIM1_default_Prescaler 0x3fff
-#define TIM1_default_Period    0xff
-#define TIM3_default_Prescaler 0x3fff
-#define TIM3_default_Period    0xff
+    //timer defaults
+    //50000 us
+    TIM1_default_Prescaler = 0x3fff,
```

Comment:
again - this looks weird in an enum - these values are unrelated in terms of "enum" meaning, they are just named similarly... moreover, there are 2 sets with identical values...

Reply:
I'm keeping the constants 1:1. Reducing the number should be in another PR.